configurations/utils.py

```diff
@@ -1,51 +1,13 @@
 import inspect
-import six
-import sys
-
-from importlib import import_module
-
-from django.core.exceptions import ImproperlyConfigured
+from functools import partial
 
 
 def isuppercase(name):
     return name == name.upper() and not name.startswith('_')
 
 
 def uppercase_attributes(obj):
-    return dict((name, getattr(obj, name))
-                for name in filter(isuppercase, dir(obj)))
-
-
-def import_by_path(dotted_path, error_prefix=''):
```

Comment:
I guess this may be in use by projects using django-configurations.
In which case it might be useful to keep and deprecate this, possibly implementing it using `import_string` if you want to cut down the amount of code.